packages/cubejs-backend-native/src/python/entry.rs

```diff
@@ -20,6 +21,15 @@ fn python_load_config(mut cx: FunctionContext) -> JsResult<JsPromise> {
     py_runtime_init(&mut cx, channel.clone())?;
 
     let conf_res = Python::with_gil(|py| -> PyResult<CubeConfigPy> {
+        let sys_path = py.import("sys")?.getattr("path")?.downcast::<PyList>()?;
```

Comment:
Let's add a small test for it. Inside `test/python.test.ts`. We can add an import in `config.py`. It should be enough to test it.

Thanks